CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ set(CMAKE_BUILD_TYPE Release)
 include(GNUInstallDirs)
 list(APPEND CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_FULL_LIBDIR})
 
+set(CUDA_MIN_VERSION 10.0)
+find_package(CUDAToolkit ${CUDA_MIN_VERSION} REQUIRED)
```

Comment:
This is already done by cudawrappers
https://github.com/nlesc-recruit/cudawrappers/blob/d4638bedf1231a1e380cd0c662e4642dda4da149/CMakeLists.txt#L38-L39
which is included on the line below. Why does it need to be repeated here?

Reply:
That line was removed in
 https://github.com/nlesc-recruit/cudawrappers/commit/7af83f5404e94955486f2b31654157155cf8c66a
Which is the version we are targetting in this PR.
The reason because that line was removed is to make the CUDAToolkit dependency more transparent.